core/vm/contracts_write_test.go

```diff
@@ -55,13 +55,21 @@ func MigrateDelegationsFn() MigrateDelegationsFunc {
 	}
 }
 
+func CalculateMigrationGasFn() CalculateMigrationGasFunc {
+	return func(db StateDB, migrationMsg *stakingTypes.MigrationMsg, homestead bool, istanbul bool) (uint64, error) {
+		return 0, nil
```

Comment:
So the MIgration gas func is not tested?

Reply:
Now tested as part of `TestEVMStaking` in `core/evm_test.go` for 0/1/2 delegations to migrate.